### PR TITLE
Rename Store REST APIs and update REST APIs version to v2

### PR DESCRIPTION
**Pull request by vithu30**
This PR contains following changes:
- Renames Store REST APIs, from `/store` to `/devportal`.
- Update the REST API version to `v2`

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/dto/APIDTO.java [hunk 1]
```diff
@@ -718,15 +718,15 @@ public void setVisibility(VisibilityEnum visibility) {
   }
 
   /**
-   * The user roles that are able to access the API in Store
+   * The user roles that are able to access the API in Devportal
    **/
```
Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/dto/APIDTO.java:722]: Shall we use 'Developer Portal'

Comment (vithu30) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/dto/APIDTO.java:722]: +1. Fixed in b45fd55

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/devportal-api.yaml [hunk 2]
```diff
@@ -2972,7 +2972,7 @@ paths:
         Get the List of API Developer Portal Alert Types.
       description: |
         This operation is used to get the list of supportd alert types for the 'subscriber' agent.
-      operationId: getStoreAlertTypes
+      operationId: getDevportalAlertTypes
```
Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/devportal-api.yaml:2975]: When it is used inside methods etc, shall we use DevPortal -- here eg: getDevPortalAlertTypes

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/devportal-api.yaml [hunk 3]
```diff
@@ -4948,9 +4948,9 @@ components:
         advertised:
           type: boolean
           example: true
-        originalStoreUrl:
+        originalDevportalUrl:
```
Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/devportal-api.yaml:4951]:
```suggestion
        originalDevPortalUrl:
```

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/data/Utils.jsx [hunk 4]
```diff
@@ -21,7 +21,7 @@ import Settings from 'Settings';
 import AuthManager from "./AuthManager";
 
 /**
- * Utility class for Store application
+ * Utility class for Devportal application
```
Comment (malinthaprasan) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/data/Utils.jsx:24]:
```suggestion
 * Utility class for Developer Portal application
```

**Review by malinthaprasan — APPROVED** [review]

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/devportal-api.yaml (hunk 2 above)
Comment (vithu30) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/devportal-api.yaml]: Fixed in 7033c14

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/devportal-api.yaml (hunk 3 above)
Comment (vithu30) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/devportal-api.yaml]: Fixed in 7033c14

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/data/Utils.jsx (hunk 4 above)
Comment (vithu30) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/data/Utils.jsx]: Fixed in 7033c14

**Pull request merged**